documentation/Tutorials/WPF/Bootstrapper.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: The Bootstrapper class
+---
+
+[Contents](Contents) [Previous](ShellView) [Next](App_Xaml)
+
+## The Bootstrapper class
+
+### Create the class
+
+Create the Bootstrapper class, make sure it is public and inherit from ``Bootstrapperbase``
+
+ ```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Caliburn.Micro
+
+namespace Caliburn.Micro.Tutorial.Wpf
+  {
+  public class Bootstrapper: BootstrapperBase
+    {
+    }
+  }
+
+  ```
+
+Note: for CSharp 9.0 it appears you do not need a using statement to ``Caliburn.Micro``.
```

Comment:
Think this is the case in this sample because Caliburn.Micro is in the namespace of the project

Reply:
It is strange and really surprised me. Microsoft changed something, but I did not read anything about it. I probably will adapt the text here a bit.

Reply:
Did a test, you are correct here. I will change the text here. The alternative would be to change all namespaces in the tutorial application. I think this is not worth the trouble. I think I may assume people know how to work with Visual Studio